.github/workflows/report-on-vulnerabilities.yaml

```diff
@@ -27,13 +27,14 @@ jobs:
       id: parse-results
       continue-on-error: true
       run: |
-        VULNS=$(cat scan.json | jq '.Results[] | has("Vulnerabilities")')
-        if echo $VULNS | grep -q 'true'; then
-          echo "Vulnerabilities found, creating issue"
-          echo "results=$(cat scan.json)" >> $GITHUB_OUTPUT
-        else
+        VULNS=$(cat scan.json | jq '.Results[] | select(.Target=="ko-app/kyverno") | length')
+        if [[ $VULNS -eq 0 ]]
+        then
           echo "No vulnerabilities found, halting"
           echo "results=nothing" >> $GITHUB_OUTPUT
+        else
+          echo "Vulnerabilities found, creating issue"
+          echo "results=$(cat scan.json | jq -c)" >> $GITHUB_OUTPUT
```

Comment:
```suggestion
          echo "has_vulns=true" >> $GITHUB_OUTPUT
```

Reply:
Something like this.

Reply:
Ok

Reply:
And `echo "has_vulns=false" >> $GITHUB_OUTPUT` in the other case.

Reply:
@eddycharly see latest changes